src/main/resources/wa-task-initiation-employment-et_englandwales.dmn

```diff
@@ -906,7 +906,7 @@ else null</text>
           <text></text>
         </inputEntry>
         <inputEntry id="UnaryTests_0jperqz">
-          <text>true</text>
+          <text></text>
```

Comment:
Since no other Task is using that column, we can just remove the ET3 Rule26 column from the DMN file all together.